ansys/dpf/core/operators/averaging/elemental_difference_fc.py

```diff
@@ -90,36 +106,35 @@ def _spec():
             map_input_pin_spec={
                 0: PinSpecification(
                     name="fields_container",
-                    type_names=['fields_container'],
                     optional=False,
                     document="""""",
                 ),
                 1: PinSpecification(
                     name="mesh",
-                    type_names=['abstract_meshed_region', 'meshes_container'],
                     optional=True,
-                    document="""The mesh region in this pin is used to perform the averaging, if there
-        is no field's support it is used""",
+                    document="""The mesh region in this pin is used to
+        perform the averaging, if there is no
+        field's support it is used""",
                 ),
                 3: PinSpecification(
                     name="scoping",
-                    type_names=['scoping', 'scopings_container'],
                     optional=True,
-                    document="""Average only on these elements, if it is scoping container, the label
-        must correspond to the one of the fields container""",
+                    document="""Average only on these elements, if it is
+        scoping container, the label must
+        correspond to the one of the fields
+        container""",
                 ),
                 10: PinSpecification(
                     name="collapse_shell_layers",
-                    type_names=['bool'],
                     optional=True,
-                    document="""The max elemental difference is taken through the different shell
-        layers if true (default is false)""",
+                    document="""The max elemental difference is taken through
+        the different shell layers if true
+        (default is false)""",
                 ),
             },
             map_output_pin_spec={
                 0: PinSpecification(
                     name="fields_container",
-                    type_names=['fields_container'],
```

Comment:
@jleonatti this is needed to work I think

Reply:
I believe this is fixed now!